training_maquetado/index.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+</head>
+
+<body>
+  <div class="title-container"> 
+    <img class="triangle" src="assets/triangle.svg">
+    <h1 class="title">Front-End Tools</h1>
+  </div> 
+
+  <div class="tech-container"> 
+        <img class="tech-icon" src="assets/react_icon.svg">
+        <h2 class="tech-title">ReactJS</h2>
+        <h2 class="tech-release">Release: 2013</h2>
+        <p class="tech-desc">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+  </div>   
+
+</body>
+
+</html>
```

Comment:
We usually leave an empty line at the end of the file so this icon do not appear on github